Author: Svante Signell <svante.signell@gmail.com>
Last-Update: Wed, 24 Sep 2014 10:36:02 UTC
Bug-Debian: http://bugs.debian.org/762673
Description: PATH_MAX is missing on Hurd so avoid this

--- a/ajax/core/ajfile.c.orig	2013-07-15 23:25:29.000000000 +0200
+++ b/ajax/core/ajfile.c	2014-09-24 11:01:42.000000000 +0200
@@ -8574,9 +8574,9 @@
 
 const AjPStr ajFileValueCwd(void)
 {
-    char cwd[PATH_MAX+1];
+    char *cwd = getcwd(NULL,0);
 
-    if(!getcwd(cwd,PATH_MAX))
+    if(!cwd)
     {
         ajStrAssignClear(&fileCwd);
 
@@ -8589,6 +8589,7 @@
     if(!ajStrSuffixC(fileCwd, SLASH_STRING))
         ajStrAppendC(&fileCwd, SLASH_STRING);
 
+    free(cwd);
     return fileCwd;
 }
 
